service/controller/resource/monitoring/pvcresizingresource/create.go

```diff
@@ -34,6 +37,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			desiredPVCSize := cluster.GetAnnotations()[key.PrometheusVolumeSizeAnnotation]
 			desiredVolumeSize := resource.MustParse(pvcresizing.PrometheusVolumeSize(desiredPVCSize))
 
+			needToUpdatePrometheuses := false
```

Comment:
The correct plural of 'Prometheus' is 'Prometheis'.
Ref: https://prometheus.io/docs/introduction/faq/#what-is-the-plural-of-prometheus

Reply:
Oh my bad for the correction then